user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -31,3 +31,75 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
 >
 > - To read more about Storage as a Service and its advantages, see [DataMiner STaaS: a game-changer for your storage needs](https://community.dataminer.services/dataminer-staas-a-game-changer-for-your-storage-needs/).
 > - As Storage as a Service is **scheduled for production in Q3 of 2023**, we are working on adding more information to this section. Check back for updates frequently!
+
+## Setup
+
+> [!IMPORTANT]
+>
+> The minimum DataMiner version to use Storage as a Service is 10.3.10 (CU1).
+> We recommend the use of the latest available version to get the latest features and performance updates.
+
+  1. Make your system **Cloud Connected** and note down the organization and coordination IDs. These can be found here: ClientTestTool>Advanced>CcaGateway>Get Global State. More information about getting cloud connected can be found [here](https://docs.dataminer.services/user-guide/Cloud_Platform/Connecting_to_cloud/Connecting_your_DataMiner_System_to_the_cloud.html).
+  1. Make sure you have at least **DataMiner CloudGateway 2.8.0** installed on the system. You can check this in the [Admin App](https://admin.dataminer.services) in the *nodes* section of your DataMiner Systems. If this isn't the case, you can immediately update it from that page. You can find more information about this [here](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)
+  1. Contact a Skyline representative to register you system to STaaS 
+  1. **Optional**: Migrate data to STaaS. To migrate your current data to STaaS please contact your Skyline representative to do so.
+  1. Edit the DB.xml, present in the Skyline DataMiner root folder, to look like this:
+
+      ```xml
+      <?xml version="1.0"?>
+      <DataBases xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.skyline.be/config/db">
+        <DataBase active="true" local="true" search="true" cloud="true" type="CloudStorage"/>
+      </DataBases>
+      ```
+
+  1. Restart your DMA and STaaS will be used.
+
+> [!IMPORTANT]
+>
+> Steps 4., 5. and 6. can only be done after registration is completed (Step 3.)
+
+> [!NOTE]
+>
+> If you experience any issues during setup or while using Storage as a Service do not hesitate to contact the Cloud Ecosystem by sending an email to: staas@dataminer.services.
+
+## Migration
+Migration to STaaS from Cassandra Cluster or Cassandra Single Node (w/o Elastic) setups is supported for all datatypes with the exception of logger tables. 
+Migration of logger tables will be supported in future versions. 
+Migration from MySQL setups is not yet supported.
+
+> [!NOTE]
+>
+> Data stored in Cassandra and Elastic databases can only be migrated to STaaS if the databases are defined in the `DB.xml` configuration file.
```

Comment:
I'm not sure if this note is really necessary. If a database is not included in DB.xml, surely that means it's not part of the DataMiner storage setup, so you could only expect it to not be included in the migration? Or possibly you mean something like this?

```suggestion
> A migration to STaaS will only include the databases DataMiner uses for its main storage. Any other Cassandra or Elasticsearch databases, which are not included in [DB.xml](xref:DB_xml), will not be migrated.
```

Reply:
well, for Elastic at least, it can be defined in DBConfiguration_xml: https://docs.dataminer.services/user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/DBConfiguration_xml.html

and if the Elastic config is defined there and not it DB.xml, the migration of Elastic data will not work

Reply:
Aha! Does that mean that it is in fact just setups with multiple Elasticsearch clusters that do not get migrated: https://docs.dataminer.services/user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/DBConfiguration_xml.html#configuring-multiple-elasticsearch-clusters?

Reply:
no, because data will not be migrated even if there is only 1 Elastic cluster, but it is defined in DBConfiguration.xml instead of DB.xml. 
there is already a story on our backlog to support this though

Reply:
Could I rephrase this like this then?

```suggestion
> Migration of an Elasticsearch database that is defined in DBConfiguration.xml instead of DB.xml (e.g. because a setup with multiple Elasticsearch clusters is used) is not yet supported.
```

Reply:
yes, looks good to me